stdlib/REPL/docs/src/index.md

```diff
@@ -135,7 +135,23 @@ julia> ; # upon typing ;, the prompt changes (in place) to: shell>
 shell> echo hello
 hello
 ```
+* Note for Windows users: Julia Shell mode does not expose windows shell commands.Hence, this will fail: 
```

Comment:
```suggestion
!!! note
    For Windows users, Julia's shell mode does not expose windows shell commands.
    Hence, this will fail:
```

Reply:
closing (?)  this pull request, in order to resubmit with your suggestions, and other small enhancements (failing test etc), in a proper branch this time (submission guidelines). Thanks for reviewing!

Reply:
You can push new commits to the existing pull request. It's much easier if you just commit the suggestions on the same branch.